[2023-01-05T10:57:05.290+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: atn_update_datalake.add_init_step_to_second_emr manual__2023-01-05T10:56:03.364986+00:00 [queued]>
[2023-01-05T10:57:05.303+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: atn_update_datalake.add_init_step_to_second_emr manual__2023-01-05T10:56:03.364986+00:00 [queued]>
[2023-01-05T10:57:05.304+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-01-05T10:57:05.304+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 3
[2023-01-05T10:57:05.304+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-01-05T10:57:05.319+0000] {taskinstance.py:1389} INFO - Executing <Task(EmrAddStepsOperator): add_init_step_to_second_emr> on 2023-01-05 10:56:03.364986+00:00
[2023-01-05T10:57:05.326+0000] {standard_task_runner.py:52} INFO - Started process 328 to run task
[2023-01-05T10:57:05.329+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'atn_update_datalake', 'add_init_step_to_second_emr', 'manual__2023-01-05T10:56:03.364986+00:00', '--job-id', '3767', '--raw', '--subdir', 'DAGS_FOLDER/market_data_analytics/my_dags/datalake_update.py', '--cfg-path', '/tmp/tmpfpz6i_4i', '--error-file', '/tmp/tmpb_mp1spy']
[2023-01-05T10:57:05.329+0000] {standard_task_runner.py:80} INFO - Job 3767: Subtask add_init_step_to_second_emr
[2023-01-05T10:57:05.396+0000] {task_command.py:371} INFO - Running <TaskInstance: atn_update_datalake.add_init_step_to_second_emr manual__2023-01-05T10:56:03.364986+00:00 [running]> on host 370ed6868bc3
[2023-01-05T10:57:05.586+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=atn
AIRFLOW_CTX_DAG_ID=atn_update_datalake
AIRFLOW_CTX_TASK_ID=add_init_step_to_second_emr
AIRFLOW_CTX_EXECUTION_DATE=2023-01-05T10:56:03.364986+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-05T10:56:03.364986+00:00
[2023-01-05T10:57:05.597+0000] {base.py:68} INFO - Using connection ID 'aws_credentials' for task execution.
[2023-01-05T10:57:05.598+0000] {connection_wrapper.py:210} INFO - AWS Connection (conn_id='aws_credentials', conn_type='aws') credentials retrieved from login and password.
[2023-01-05T10:57:05.667+0000] {emr.py:104} INFO - Adding steps to j-1Z5DDUK8PTRNA
[2023-01-05T10:57:06.761+0000] {emr.py:117} INFO - Steps ['s-277NCP4AH3TAQ', 's-2VP15JGSH21WV', 's-2MSF0WHR496LZ', 's-2DE8K0CJILZJ6', 's-3M4I8P3UXLFCI', 's-3P43BLD0L52KS', 's-3700MID0QJC4T'] added to JobFlow
[2023-01-05T10:57:06.817+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=atn_update_datalake, task_id=add_init_step_to_second_emr, execution_date=20230105T105603, start_date=20230105T105705, end_date=20230105T105706
[2023-01-05T10:57:06.870+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-05T10:57:06.969+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
